packages/design/tailwind/css/components.css

```diff
@@ -442,3 +442,9 @@ input[type='file' i] {
   @apply gi-mt-2;
 }
 /* TextArea */
+
+/* Chip */
+.gi-chip {
+  @apply gi-flex gi-items-center gi-gap-1 gi-pl-1.5 gi-pr-1 gi-py-0.5 gi-bg-gray-100 gi-w-fit gi-rounded-sm gi-cursor-default gi-text-xs hover:gi-bg-gray-200 focus:gi-bg-gray-200 focus:gi-outline focus:gi-outline-gray-950 focus:gi-outline-[2px] focus:gi-shadow-[inset_0_0_0_1px_white,0_0_0_5px_var(--gieds-color-yellow-400)];
```

Comment:
Is this a different focus style than we have in the focus classes ? If it's not, can we use the focus classes instead ?

Reply:
I had to use different focus styles for the Chip, so I couldn't use the classes created.